src/v2/mod.rs

```diff
@@ -72,3 +72,18 @@ impl Zitadel {
 		self.domain.join(endpoint).context("Invalid relative path for the url")
 	}
 }
+
+/// Format a request for logging purposes; this *must* omit any
+/// secrets or otherwise sensitive information
+fn format_request_for_log(request: &Request) -> String {
+	format!(
+		"{} {}, headers: {:?}",
+		request.method(),
+		request.url(),
+		request
+			.headers()
+			.into_iter()
+			.filter(|(name, _)| *name != "authorization")
```

Comment:
If we want to redact manually we probably want to normalize the header (to lower case for example). Otherwise it can skip `Authorization`